src/openrct2/peep/Peep.h

```diff
@@ -32,7 +33,7 @@ namespace GameActions
 {
     class Result;
 }
-using ParkEntranceIndex = uint8_t;
+using ParkEntranceIndex = TIdentifier<uint8_t, std::numeric_limits<uint8_t>::max(), struct ParkEntranceIndexTag>;
```

Comment:
It's a bit odd that this is defined in two places. I think it's better to just include Entrance.h in this file.

Reply:
I avoided including it, Entrance.h is heavy due to Map.h

Reply:
Include with "../ride/Station.h", and forward declare `struct CoordsXYE`, and you can remove Map.h.

Reply:
perhaps we should have a header that is just all the identifier types.

Reply:
Makes sense, ill revert the commit